### ISSUE TITLE
Does the YOLOV11 segmentation task support PASCAL VOC format in the dataset? I want to use mask as the truth value instead of polygon contour as the truth value

### ISSUE THREAD

**Issue by litengfeigithub** [ultralytics/ultralytics#17915]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Does the YOLOV11 segmentation task support PASCAL VOC format in the dataset? I want to use mask as the truth value instead of polygon contour as the truth value

**Comment by UltralyticsAssistant** [comment]
👋 Hello @litengfeigithub, thank you for your interest in Ultralytics 🚀! We recommend checking out our [Docs](https://docs.ultralytics.com), where you'll find helpful information about various modes and tasks, including segmentation. You may also find answers in the [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage guides.

To address your query regarding the PASCAL VOC format and mask support for YOLOv11 segmentation tasks, an Ultralytics engineer will assist you shortly. In the meantime, for the community to better assist you, we recommend providing further details or examples of your dataset setup to offer clarity.

If this involves a 🐛 Bug or unexpected behavior, please include a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us investigate your issue effectively.

For a deep dive into training or dataset customization ❓ Questions, please share as much information as possible, such as your dataset's structure, sample images, or other specifics. Don’t forget to verify you're following the best practices outlined in our [Training Guides](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best! For real-time help or discussions, you can connect with others on [Discord](https://discord.com/invite/ultralytics) 🎧, [Discourse](https://community.ultralytics.com), or even our [Subreddit](https://reddit.com/r/Ultralytics).

## Upgrade

To ensure you’re using the latest and most reliable version, upgrade the `ultralytics` package with the command below. This helps resolve issues that may have already been addressed in recent updates:

```bash
pip install -U ultralytics
```

Ensure all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) are installed in a [Python>=3.8](https://www.python.org/) environment, along with [PyTorch>=1.8](https://pytorch.org/get-started/locally/).

## Environments

You can test YOLO models in a variety of dependable environments (complete with CUDA/CUDNN and necessary dependencies):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. [Docker Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI Tests](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) are passing. These nightly tests validate YOLO modes and tasks across macOS, Windows, and Ubuntu platforms.

🚀 We appreciate your contribution to the Ultralytics community and your patience while we assist you further!

**Comment by Y-T-G** [comment]
No. It's not supported.

**Comment by litengfeigithub** [comment]
> No. It's not supported.
Thank you. If I want to train segmentation tasks with YOLOV11, I need to convert the dataset from Pascal VOC format to CocoSeg8 format？

**Comment by litengfeigithub** [comment]
thank you very much

**Comment by litengfeigithub** [comment]
> YOLO Segmentation format
> 
> https://docs.ultralytics.com/datasets/segment/

I want to load a pre trained model. Where can I set it up? It's during model initialization
model = YOLO('/ltf/ultralytics-oven/ultralytics/cfg/models/11/yolo11-segl.yaml', task="segment").load("./yolo11l-seg.pt")
Set here, still in the following parameter pretrained=True, # (boolean | str) whether to use a pre trained model (boolean) or a model loaded with weights from it (str)
#Pretrained="./yolo11l-seg.pt", what is the setting for # (boolean | str)
![Uploading 1111.png…]()
![Uploading 2222.png…]()

**Comment by Y-T-G** [comment]
You can load the pt file using `YOLO()` directly. `pretrained` doesn't do anything if it is True or False.
